### PR TITLE
Add documentation for the OHLCV API

### PR DESCRIPTION
**Pull request by pushkar-g**
Signed-off-by: Pushkar Gupte <pushkar@bravenewcoin.com>

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -1124,80 +1148,32 @@ paths:
...
+            | GWA        |Asset Id   |
+
           required: false
           schema:
             type: string
             format: uuid
         - name: startAfter
           in: query
-          description: Retrieve all global weighted average OHLCV after this particular id
+          description: Retrieve market weighted average OHLCV by a particular id provided.
```
Comment (alphafoobar) [inline openapi.yaml]:
```suggestion
          description: Retrieve OHLCV for the indexId or indexType required, starting after this date time.
```

File: openapi.yaml [hunk 2]
```diff
@@ -1208,12 +1184,12 @@ paths:
...
           schema:
             type: integer
-            default: 10
+            default: 200
             minimum: 1
             maximum: 2000
       responses:
         '200':
-          description: 'The open, high, low, close and volume by asset'
+          description: 'The market weighted average OHLCV by market'
```
Comment (alphafoobar) [inline openapi.yaml]:
```suggestion
           description: 'The open, high, low, close and volume.'
```

**Review by alphafoobar — APPROVED** [review]
Looks OK, so we need to provide:

1. Latest OHLCV
2. Historic queryable OHLCV (starts after ts)

And support:

1. Query by IndexId
2. Query by index type

**Review by suryakakanadan — APPROVED** [review]
Looks OK

**Review by zhangyi733 — APPROVED** [review]

**Pull request merged**